back/test/integration/TestTechMapsApi.js

```diff
@@ -63,26 +62,331 @@ describe("API", () => {
       expect(res).to.have.status(404);
     });
 
-    it.skip("should bump up version when modified techmap put", async() => {
+    it("should bump up version when modified techmap put", async () => {
       // todo: make sure all fields are checked.
+      const id = newTechMapId();
+
+      const newTechMap = {
+        id,
+        name: "Хліб Французький (КХ)",
+        units: [1, 6],
+        steps: [
+          {
+            id: "STP-XXX-YYY-ZZZ-YY0",
+            name: "Замішування",
+            ingredients: [
+              {
+                ingredientId: "ING-XXX-YYY-ZZZ-YY0",
+                countByUnits: new Map([[1, 292], [6, 1752]])
```

Comment:
"new Map([[1, 292], [6, 1752]])" --> I would like to see real JSON data, I have no idea how this Map is serialized.

Reply:
It appears that JSON's stringify and parse can't handle maps. Discussion is required.